pymc/gp/gp.py

```diff
@@ -148,18 +148,23 @@ class Latent(Base):
     def __init__(self, *, mean_func=Zero(), cov_func=Constant(0.0)):
         super().__init__(mean_func=mean_func, cov_func=cov_func)
 
-    def _build_prior(self, name, X, reparameterize=True, jitter=JITTER_DEFAULT, **kwargs):
+    def _build_prior(
+        self, name, X, num_outputs=1, reparameterize=True, jitter=JITTER_DEFAULT, **kwargs
+    ):
         mu = self.mean_func(X)
         cov = stabilize(self.cov_func(X), jitter)
         if reparameterize:
             size = np.shape(X)[0]
             v = pm.Normal(name + "_rotated_", mu=0.0, sigma=1.0, size=size, **kwargs)
```

Comment:
As a starting point, you'll need to make `v` here have size=(n_outputs, X.shape[0]) (or transpose of that).  To check, you can draw some samples from the prior predictive with multiple outputs.  

You'd have to do the same for the case `reparameterize=False`.  But honestly, I think that is so rarely used it'd be just fine to just raise a `NotImplementedError` if `n_outputs > 1`.

Reply:
Thanks for your comments and suggestions. If we want to make the size of `v` equal to (n_outputs, X.shape[0]) or its transpose, maybe we should replace `pm.Normal` with `pm.MvNormal` of zero mean and identity cov

Reply:
glad I could help, looking forward to it

Reply:
Does work 🍾 
![image](https://github.com/user-attachments/assets/bbee07ef-863b-4d91-babf-ad4d79f2072a)